scm/const.go

```diff
@@ -67,6 +67,8 @@ func (a Action) String() (s string) {
 		return "synchronized"
 	case ActionMerge:
 		return "merged"
+	case ActionEdit:
```

Comment:
@bradrydzewski how do you feel about adding this new action, it does seem to be referenced in some of the drivers.

Reply:
We should not need to add a new action here.  If this refers to updating pull request code we should use the `synchronized` action (which is the abstracted term we use for all providers) or if this refers to updating metadata (tltle, description, status, etc) we should use the `update` action.

See some of the existing implementations for prior art:
https://github.com/drone/go-scm/blob/master/scm/driver/github/webhook.go#L159:L160